monai/bundle/scripts.py

```diff
@@ -172,3 +224,88 @@ def verify_metadata(
         logger.info(re.compile(r".*Failed validating", re.S).findall(str(e))[0] + f" against schema `{url}`.")
         return
     logger.info("metadata is verified with no error.")
+
+
+def verify_net_in_out(
+    net_id: Optional[str] = None,
+    meta_file: Optional[Union[str, Sequence[str]]] = None,
+    config_file: Optional[Union[str, Sequence[str]]] = None,
+    device: Optional[str] = None,
+    p: Optional[int] = None,
+    n: Optional[int] = None,
+    any: Optional[int] = None,
+    args_file: Optional[str] = None,
+    **override,
+):
+    """
+    Verify the input and output data shape and data type of network defined in the metadata.
+    Will test with fake Tensor data according to the network args with id:
+    input data:
```

Comment:
Could you remove the examples in line 234-249, please put one command line example instead

Reply:
Sure, updated in the latest commit.

Thanks.